src/snowflake/snowpark/modin/plugin/_internal/indexing_utils.py

```diff
@@ -96,6 +96,7 @@
     "Must have equal len keys and value when setting with an iterable"
 )
 LOC_SET_ITEM_EMPTY_ERROR = "The length of the value/item to set is empty"
+_LOC_SET_NON_TIMEDELTA_TO_TIMEDELTA_ERROR = "Snowpark pandas does not yet support assigning timedelta values to an existing column."
```

Comment:
nit - Why is only this constant an internal variable?

Reply:
Outside of this file, it's only used in tests, so it's not part of the interface of this module.